doc/website/FAQ.md

```diff
@@ -2,10 +2,59 @@
 
 In this document are tips and hints documented which can help for troubleshooting on RouDi.
 
-## Does iceoryx run in a docker environment
+## Does iceoryx run in a docker environment?
 
 Yes. Take a look at the [icedocker example](../../iceoryx_examples/icedocker/)
 
+## How can I find out if RouDi is running?
+
+RouDi uses a file locking machanism to ensure that only one RouDi instance is running at a time. For that RouDi
+creates and locks `/tmp/roudi.lock`. The file exists also also when RouDi is not running. Try locking this file,
+if this fails RouDi is running.
+
+On the command line do
+
+```console
+flock -n /tmp/roudi.lock echo "RouDi is not running"
+```
+
+Or for example in Python
+
+```python
+try:
+    fcntl.flock(roudi_lock.fileno(), fcntl.LOCK_EX | fcntl.LOCK_NB)
+except BlockingIOError as exception:
+    raise RuntimeError("RouDi already running.") from exception
+finally:
+    fcntl.flock(roudi_lock.fileno(), fcntl.LOCK_UN)
+```
+
+## Missing samples in a publish subscribe scenario
+
+Lost samples are not acceptable for your system? Is your publisher sending at a higher frequency than the subscribers are taking the samples?
+
+Either
+
+* Make sure that the receiving frequency is higher than the publishing one
+
+or
+
+* Use the [blocking publisher feature](../../iceoryx_examples/iceoptions/)
```

Comment:
I think we should not encourage to use the blocking publisher or at least put a big disclaimer that other subscriber will not receive samples while the publisher is blocked.